src/System.Net.NameResolution/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
There's a lot of unlocking going on; but that makes the builds unstable in the future (as new packages are release the lock files get edited to say they happily grabbed newer content).  Can we go back to locked?